modules/custom/social_language/social_language.install

```diff
@@ -124,5 +124,12 @@ YAML;
  * Give translate permissions to content managers.
  */
 function social_language_update_8002() {
-  _social_language_set_permissions();
+  // Check if the permissions for site managers have been restricted.
+  if ($role = Role::load('sitemanager')) {
+    if ($role->hasPermission('create content translations')) {
+      $permissions = _social_language_get_permissions('contentmanager');
```

Comment:
We provide contentmanager permissions here based on settings for a sitemanager?

Reply:
Yes, this permission was added in the installation for site managers. If they removed the permission they probably do not want them added for content managers.